Added support for labels and existing PR check (v2 from master)

vitalyal opened the pull request.

thomaseizinger commented on CHANGELOG.md:
```diff
@@ -22,6 +24,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Initial release!
-
-[Unreleased]: https://github.com/thomaseizinger/create-pull-request/compare/1.0.0...HEAD
```
As mentioned in the other PR, this needs to stay.

I'll update the master branch on my repo and then you can rebase / merge.

thomaseizinger reviewed (changes requested):
Unfortunately, not all the checks are passing currently (even ignoring the "reviewers" check): https://github.com/vitalyal/create-pull-request/actions/runs/1058829910

vitalyal commented:
I've rebased, updated changelog and fixed CI (the only case failing is about reviewers) https://github.com/vitalyal/create-pull-request/actions/runs/1059421040

thomaseizinger commented:
> I've rebased, updated changelog and fixed CI (the only case failing is about reviewers) https://github.com/vitalyal/create-pull-request/actions/runs/1059421040

Thank you! I'll release a new version soon!

thomaseizinger commented:
Released as 1.2.0.
